bigquery_etl/shredder/config.py

```diff
@@ -363,6 +366,11 @@ def fields(self) -> tuple[str, ...]:
     context_id_target(
         table="contextual_services_stable.quicksuggest_impression_v1"
     ): CONTEXTUAL_SERVICES_SRC,
+    # client association ping
+    DeleteTarget(
+        table="firefox_desktop_stable.fx_accounts_v1",
+        field="metrics.string.client_association_uid",
+    ): FXA_UNHASHED_SRC,
```

Comment:
Just want to confirm: there's no issue with continuing to do the regular client id deletion on this ping as well right?

Reply:
Yes we should keep shredding this per regular Glean deletion-requests.
BTW I now realized that if we extend shredding via FxA uids to all the pings at some point, then we should merge DeleteTargets for tables involved to not overly increase number of queries.